static/js/context-menu.js

```diff
@@ -1208,6 +1211,10 @@ var Hover = new (class Hover {
       return this.deactivate();
     }
```

Comment:
And if we're to move the above into `updateHoverState`, it would be nice to move this there as well, to keep the ordering.